tasks/portfolio_intake.py

```
@@ -0,0 +1,154 @@
+def map_initiative_to_task(initiative: Any) -> Task:
+    """Build a Task from a portfolio Initiative dataclass."""
+    title = getattr(initiative, "title", "Portfolio initiative")
+    description = getattr(initiative, "description", "") or ""
+    horizon = getattr(initiative, "horizon", None)
+    horizon_val = horizon.value if hasattr(horizon, "value") else str(horizon or "unscheduled")
+    source = getattr(initiative, "source", "manual") or "manual"
+    wsjf = getattr(initiative, "wsjf", 0.0)
+    if callable(wsjf):
+        try:
+            wsjf = wsjf()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)